sconfig/shared/src/test/scala/org/ekrich/config/impl/ConfigFormatOptionsTest.scala

```diff
@@ -7,7 +7,7 @@ import org.ekrich.config.{
   ConfigRenderOptions,
   ConfigFormatOptions
 }
-
+// tests for new (since fork from lightbend) features of Rendering
```

Comment:
I am not following here why the tests are removed. Are the commits in this list or is it newer? https://github.com/ekrich/sconfig/issues/29

Please provide a link with explanation at the top of this PR. Thanks.

Reply:
moved to another file,
one for retaining or fixing behaviour with all new features disabled
and second one for every new feature activated